client/src/App.js

```diff
@@ -32,7 +32,7 @@ function App() {
           <Route path="/choice">
             <Choice />
           </Route>
-          <Route path="/tour">
+          <Route exact path="/tour/:tourname">
```

Comment:
Are you sure you need an exact in front of a variable path?
```suggestion
          <Route path="/tour/:tourname">
```

Reply:
changed this to a path which is notexact